README.md

```diff
@@ -34,7 +34,9 @@ You should set these up before attempting to install the repo.
 
 7. In the `next-build` directory, run `yarn setup` to pull initial built assets from the `vets-website` repo. This will grab a bunch of files from a vets-website S3 bucket and place them into the appropriate `public/` folders.
 
-8. Run `yarn dev`.
+8. In your `envs/.env.local` file, uncomment `FEATURE_NEXT_BUILD_CONTENT_ALL=true`.
```

Comment:
Thinking this guidance might be too broad; but definitely we should add a step here to show how to get data to load.

Reply:
I think this is OK for now. The feature flag implementation can use some more thought. This is good for now..